showcase/identity_test.go

```diff
@@ -23,134 +23,139 @@ import (
 	showcasepb "github.com/googleapis/gapic-showcase/server/genproto"
 	"google.golang.org/api/iterator"
 	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/fieldmaskpb"
 )
 
 var identity *showcase.IdentityClient
+var identityREST *showcase.IdentityClient
```

Comment:
FWIW, I would find it helpful to include a comment that these are initialized in `main_test.go`. I was confused how these differed until I saw that.

Reply:
Fair enough! Done (and applied to all such `var`).